src/libs/actions/Policy.js

```diff
@@ -752,6 +753,73 @@ function clearAddMemberError(policyID, memberEmail) {
 }
 
 /**
+* Checks if we have any errors stored within the POLICY_MEMBER_LIST.  Determines whether we should show a red brick road error or not
+ * Data structure: {email: {role:'bla', errors: []}, email2: {role:'bla', errors: [{1231312313: 'Unable to do X'}]}, ...}
+ * @param {Object} policyMemberList
+ * @returns {Boolean}
+ */
+function hasPolicyMemberError(policyMemberList) {
+    return _.some(policyMemberList, member => !_.isEmpty(member.errors));
+}
+
+/**
+ * @param {String} value
+ * @returns {String}
+ */
+function capitalizeFirstLetter(value) {
+    return value.charAt(0).toUpperCase() + value.slice(1);
+}
+
+/**
+ * Generate a policy name based on an email and policy list.
+ * @param {String} email
+ * @param {Array} polices
+ * @returns {String}
+ */
+function generateDefaultWorkspaceName(email, polices = null) {
+    let userPolicies = null;
+    if (!polices) {
+        userPolicies = allPolicies;
+    } else {
+        userPolicies = polices;
+    }
```

Comment:
Hmm, we don't want this code since it is just specific to tests

Reply:
+1